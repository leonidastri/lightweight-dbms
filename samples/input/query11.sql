Select DISTINCT S.B, B.E, R.G from Sailors S, Boats B, Reserves R WHERE S.B < B.D AND B.E = R.G ORDER BY S.B, B.E;