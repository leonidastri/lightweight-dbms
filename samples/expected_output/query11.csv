100,1,1
100,2,2
100,3,3